plugin/src/main/java/org/jenkinsci/plugins/workflow/cps/CpsThreadDumpAction.java

```diff
@@ -108,13 +118,47 @@ public CpsThreadDump getThreadDump() {
 
         @Override public void addContents(Container container) {
             container.add(new Content("nodes/master/pipeline-thread-dump.txt") {
```

Comment:
Is it worth updating the display name and file name to be more generic, and/or moving this to a distinct class, or should we just keep it? Users may be used to the current display name, and there may be some CloudBees-internal automation which expects the current file name, but I am not sure.

Reply:
Unclear to me how this related to the existing `pipeline-timings.txt`. Should we consolidate?

Reply:
One is for completed builds, one is for running builds. If I was going to rename both of them, I would do this:
* "Timing data about recently completed Pipeline builds"/`pipeline-timings.txt` would become "Recently completed Pipeline builds"/`pipeline-recent-builds.txt`
* "Thread dumps of running Pipeline builds"/`pipeline-thread-dump.txt` would become "Running Pipeline builds"/`pipeline-running-builds.txt`

We could combine them into a single "Pipeline builds"/`pipelines.txt` component, but I think keeping running and completed builds seems clearest.

Reply:
> I would do this

Seems reasonable. Anyway, this is fine as is, just trying to clarify.

Reply:
I did some GitHub searches for usage of these exact strings and checked with CloudBees support about support bundle automation and didn't find anything problematic, so I will go ahead and update the names here.

Reply:
See [e8af1ac](https://github.com/jenkinsci/workflow-cps-plugin/pull/916/commits/e8af1ac4ede9f3b07ae412e7747ee83ca77bfd46).